integration-libs/cds/src/schematics/add-cds/index_spec.ts

```diff
@@ -94,42 +99,72 @@ describe('Spartacus CDS schematics: ng-add', () => {
         .toPromise();
     });
 
-    it('should not create the feature module', () => {
-      const featureModule = appTree.readContent(spartacusCdsFeatureModulePath);
-      expect(featureModule).toBeFalsy();
+    it('should not create any of the feature modules', () => {
+      expect(appTree.exists(featureModulePath)).toBeFalsy();
     });
-    it('should not add the feature to the feature module', () => {
-      const spartacusFeaturesModule = appTree.readContent(
-        'src/app/spartacus/spartacus-features.module.ts'
-      );
-      expect(spartacusFeaturesModule).toMatchSnapshot();
+
+    it('should install necessary Spartacus libraries', () => {
+      const packageJsonContent = appTree.readContent('package.json');
+      const dependencies = JSON.parse(packageJsonContent).dependencies;
+
+      for (const toAdd in peerDependencies) {
+        if (!dependencies.hasOwnProperty(toAdd)) {
```

Comment:
This guard should check `peerDependencies`. Using `dependencies` here makes the test impossible to fail ;)

Additionally, we also need to skip assertion for the Skipped libs (core spartacus libs array could be useful here)

Below the version that I think should be correct:
```
const dependencies = {
        ...JSON.parse(packageJsonContent).dependencies,
        ...JSON.parse(packageJsonContent).devDependencies,
      };

      for (const toAdd in peerDependencies) {
        if (
          !peerDependencies.hasOwnProperty(toAdd) ||
          !CORE_SPARTACUS_SCOPES.includes(toAdd)
        ) {
          continue;
        }
        // TODO: after 4.0: use this test, as we'll have synced versions between lib's and root package.json
        // const expectedVersion = (peerDependencies as Record<
        //   string,
        //   string
        // >)[toAdd];
        const expectedDependency = dependencies[toAdd];
        expect(expectedDependency).toBeTruthy();
        // expect(expectedDependency).toEqual(expectedVersion);
      }
```

Reply:
Done